docs/content_management/pages/react_app_block.md

```diff
@@ -78,23 +68,20 @@ Parameters passed as props must be converted so that they can be used as the con
 
 ## Create React App block
 
-In the following example, you learn how to create `Calculator` React App block.
+In the following example, you learn how to create the `Calculator` React App block [configured in the previous section's example](#react-app-block-configuration).
 
 ### Configure React App Block
 
-First, create a .jsx file which describes your component.
+First, install the react.
```

Comment:
```suggestion
First, install the React.
```

Reply:
I guess it should be capitalized but I may be wrong :)

Reply:
I'm agree with this being a proper name.

```suggestion
First, install React.
```